lib/worker.js

```diff
@@ -15,18 +15,28 @@ const eslintPathLocal = Path.join(FS.realpathSync(Path.join(__dirname, '..')), '
 let eslint = null
 let prefixPath = null
 
-function find(startDir, name) {
-  let filePath
+function find(startDir, name, limit) {
+  const filePaths = [];
   const chunks = startDir.split(Path.sep)
+
   while (chunks.length) {
-    filePath = Path.join(chunks.join(Path.sep), name)
+    const filePath = Path.join(chunks.join(Path.sep), name)
+
     try {
       FS.accessSync(filePath, FS.R_OK)
-      return filePath
+      filePaths.push(filePath)
+
+      // Stop searching once search limit is reached
+      if (limit && filePaths.length >= limit) {
+        break
+      }
     } catch (_) { }
+
     chunks.pop()
   }
-  return null
+
+  // Return null when nothing found to keep previous find() behaviour
```

Comment:
Only nitpick I have would be to remove this comment, there isn't really a need for it in the code since somebody would have to dig through the history to see what it meant anyway :wink:.